tracer/test/Datadog.Trace.Tests/Util/Delegates/FuncInstrumentationTests.cs

```diff
@@ -471,7 +471,7 @@ public async Task Async1Test()
                                                  onDelegateAsyncEnd: async (sender, returnValue, exception, state) =>
                                                  {
                                                      Interlocked.Increment(ref value).Should().Be(4);
-                                                     await Task.Delay(100).ConfigureAwait(false);
+                                                     await Task.Delay(500).ConfigureAwait(false);
```

Comment:
Is this test flaky? I'm not sure why this needs to be increased? 🤔

Reply:
Reverted.